server/src/main/java/org/apache/druid/server/compaction/DataSourceCompactibleSegmentIterator.java

```diff
@@ -491,10 +491,9 @@ static List<Interval> filterSkipIntervals(Interval totalInterval, List<Interval>
         remainingStart = skipInterval.getEnd();
       } else {
         // Ignore this skipInterval
-        log.warn(
+        log.debug(
```

Comment:
@kfaraz Should we wrap this in `log.isDebugEnabled()` since we have `new Interval(remainingStart, remainingEnd)` as one of the arguments?

My understanding is that a new instance of Interval will be created even if debug logging isn't enabled.

Reply:
That's true. But this is a rare enough scenario that the creation of the superfluous Interval object will not be costly.

Reply:
Fair enough, thanks for the info!